cloudformation/ad/ad-integration.yaml

```diff
@@ -561,7 +565,8 @@ Resources:
                   response_data['Message'] = 'Resource creation successful!'
                   physical_resource_id = create_physical_resource_id()
                   ds.reset_user_password(DirectoryId=directory_id, UserName='ReadOnlyUser', NewPassword=read_only_password)
-                  ds.reset_user_password(DirectoryId=directory_id, UserName=user_name, NewPassword=user_password)
+                  for name in user_names.split(","):
```

Comment:
I suggest to sanitize user_names by trimming it to avoid the risk of having user names with blank spaces.

The reason why I'm suggesting this here in Python code but not in the bash code above is because in the bash code you implicitly sanitized it using sed replacement within bash arrays.